src/main/resources/modules/uti/abx_tx.json

```
@@ -0,0 +1,1080 @@
+          "code": 1648756,
+          "display": "nitrofurantoin, macrocrystals 25 MG / nitrofurantoin, monohydrate 75 MG Oral Capsule",
+          "value_set": ""
+        }
+      ],
+      "reason": "uti",
+      "name": "Nitro 7 day",
+      "assign_to_attribute": "UTI_Tx",
+      "prescription": {
+        "dosage": {
+          "amount": 1,
+          "frequency": 2,
+          "period": 1,
+          "unit": "days"
+        },
+        "duration": {
+          "quantity": 7,
+          "unit": "days"
+        }
+      },
+      "direct_transition": "ED Check"
+    },
+    "Fosfo": {
+      "type": "MedicationOrder",
+      "codes": [
+        {
+          "system": "RxNorm",
+          "code": 808917,
+          "display": "fosfomycin 3000 MG Powder",
+          "value_set": ""
+        }
+      ],
+      "reason": "uti",
+      "name": "Fosfo",
+      "prescription": {
+        "dosage": {
+          "amount": 1,
+          "frequency": 1,
+          "period": 1,
+          "unit": "days"
+        },
+        "duration": {
+          "quantity": 1,
+          "unit": "days"
+        }
+      },
+      "assign_to_attribute": "UTI_Tx",
+      "direct_transition": "ED Check"
+    },
+    "Sulfa 3 day": {
+      "type": "MedicationOrder",
+      "codes": [
+        {
+          "system": "RxNorm",
+          "code": 198335,
+          "display": "sulfamethoxazole 800 MG / trimethoprim 160 MG Oral Tablet",
+          "value_set": ""
+        }
+      ],
+      "reason": "uti",
+      "name": "Sulfa 3 day",
+      "prescription": {
+        "dosage": {
+          "amount": 1,
+          "frequency": 1,
+          "period": 2,
+          "unit": "days"
+        },
+        "duration": {
+          "quantity": 3,
+          "unit": "days"
+        }
+      },
+      "direct_transition": "ED Check",
+      "assign_to_attribute": "UTI_Tx"
+    },
+    "Sulfa 7 day": {
+      "type": "MedicationOrder",
+      "codes": [
+        {
+          "system": "RxNorm",
+          "code": 198335,
+          "display": "sulfamethoxazole 800 MG / trimethoprim 160 MG Oral Tablet",
+          "value_set": ""
+        }
+      ],
+      "reason": "uti",
+      "name": "Sulfa 7 day",
+      "prescription": {
+        "dosage": {
+          "amount": 1,
+          "frequency": 1,
+          "period": 2,
+          "unit": "days"
+        },
+        "duration": {
+          "quantity": 7,
+          "unit": "days"
+        }
+      },
+      "direct_transition": "ED Check",
+      "assign_to_attribute": "UTI_Tx"
+    },
+    "Sulfa 10 day": {
+      "type": "MedicationOrder",
+      "codes": [
+        {
+          "system": "RxNorm",
+          "code": 198335,
+          "display": "sulfamethoxazole 800 MG / trimethoprim 160 MG Oral Tablet",
+          "value_set": ""
+        }
+      ],
+      "reason": "uti",
+      "name": "Sulfa 10 day",
+      "prescription": {
+        "dosage": {
+          "amount": 1,
+          "frequency": 1,
+          "period": 2,
```

I think the `frequency` and `period` are flipped for the sulfa drugs in this module. Looks like everything is "twice daily" and all of the other states show frequency:2 period:1, but the 3 sulfa drugs show frequency:1 period:2
```suggestion
          "frequency": 2,
          "period": 1,
```